content/community/engineering_guidelines.md

```diff
@@ -50,6 +50,12 @@ Using [zenhub.io](https://zenhub.io) allows us to link PRs to issues without nec
 
 Pretty self-explanatory: no self-merged PRs without prior approval from maintainers. Reviews are everyone's responsibility and should focus on tests and documentation as much as implementation.
 
+However, it should taken care that the PR review process does not take too long as it is easy to get lost in the reviews:
+
+* A PR which is marked as 'done' (i.e. not marked as 'work-in-progress') should be merged or rejected with 3 business days.
+* There should be at most 2 review turnarounds (i.e. reviewer requests changes - author makes changes - reviewer request final changes - author makes changes - reviewer accepts / rejects).
+* A reviewer should be actively selected by the author immediately after creating the PR, e.g. asking on IRC or mailing list.
```

Comment:
I think its better for the moment to do this manually, because its more about sync vs. async. If you can ask one directly and he agrees, its better than when the assignment mail is somewhere buried in the mail backlog.